packages/template-parser/src/index.ts

```diff
@@ -49,10 +49,11 @@ const KEYS: VisitorKeys = {
   PropertyRead: ['receiver'],
   Template: ['templateAttrs', 'children', 'inputs'],
   BindingPipe: ['exp'],
-  DeferredBlock: ['children'],
+  DeferredBlock: ['children', 'placeholder', 'loading', 'error'],
```

Comment:
Without this angular-eslint won't parse `@placeholder`, `@loading` or `@error` nodes